app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    people_list = [Person(person["name"], person["age"]) for person in people]
+
+    for person in people:
+        person_instance = Person.people[person["name"]]
+
+        if "husband" in person and person["husband"] is not None:
```

Comment:
Use just dict.get() method in the condition, instead of this long one

Reply:
Hi, @LiudmylaKulinchenko!
Could you please elaborate on this one?